### PR TITLE
Baby step toward in-editor generation

### PR DESCRIPTION
**Pull request by eob**

### CODE REVIEW

File: tests/steamship_tests/endpoints/test_game_editor_endpoints.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+def test_generate_suggestion(
+    invocable_handler_with_client: Tuple[
+        Callable[[str, str, Optional[dict]], dict], Steamship
+    ]
+):
+    invocable_handler, client = invocable_handler_with_client
+    task_dict = invocable_handler(
+        "POST", "generate_suggestion", {"field_name": "narrative_voice"}
+    )
+    block = Block.parse_obj(task_dict.get("data"))
```
Comment (douglas-reid) [inline tests/steamship_tests/endpoints/test_game_editor_endpoints.py:40]: note: there is now a `blocks_from_invoke` in the SDK. not sure if that would help or not, as I forget the contract of `invocable_handler` at this point.

**Review by douglas-reid — APPROVED** [review]
Excited to see this in action. Just a bunch of non-blocking questions.

File: src/generators/editor_suggestions/editor_suggestion_generator.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+
+class EditorSuggestionGenerator:
+    def generate_editor_suggestion(
+        self, field_name: str, unsaved_server_settings: dict, context: AgentContext
+    ) -> Task:
+        server_settings = get_server_settings(context)
+        generator = get_story_text_generator(context)
+
+        prompt = PROMPTS.get(field_name)
```
[the remark this answers is not in the document]
Comment (eob) [reply src/generators/editor_suggestions/editor_suggestion_generator.py:85]: Yeah I agree -- I think there's a lot of "config validation / consolidation" that we'd benefit from.

File: src/generators/editor_suggestions/editor_suggestion_generator.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+- silly, like a Cartoon Network show
+- parody, in the style of an HBO show
+- dramatic, with a tinge of danger
+- gritty, high contrast and real
+- film noir, with a romantic tinge of hard-boiled mystery
+
+Suggestion:
+
+-""",
+    "adventure_background": """I need help! Write a few notes for a director about an scene in a short story.
```
[the remark this answers is not in the document]
Comment (eob) [reply src/generators/editor_suggestions/editor_suggestion_generator.py:36]: It seems OK? But it's not tested. I'm not sure how others tend to iterate on prompts but sometimes I'm bit uneasy how little time I spend on them compared to coding elsewhere

**Pull request merged**